src/System.Threading.Thread/tests/MTAMain/MTAMain.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+
+namespace MTAMain
+{
+    internal static class MTAMain
+    {
+        [MTAThread]
+        static int Main(string[] args)
+        {
+            string mode = args[0];
+            int retValue = 1;
```

Comment:
nit: It would be easier to read if 1/0 were named constants (failure/success, or the like)